content/en/blog/_posts/2017-05-00-Kubernetes-Security-Process-Explained.md

```diff
@@ -3,6 +3,9 @@ title: " Dancing at the Lip of a Volcano: The Kubernetes Security Process - Expl
 date: 2017-05-18
 slug: kubernetes-security-process-explained
 url: /blog/2017/05/Kubernetes-Security-Process-Explained
+author: >
+  Brandon Philips (CoreOS),
+  Jess Frazelle (Google)
 ---
 _Editor's note: Today’s post is by&nbsp; __Jess Frazelle of Google and Brandon Philips of CoreOS about the Kubernetes security disclosures and response policy.__ &nbsp;_  
```

Comment:
I'd drop this note. We now just credit the person and optionally their organization.